src/core/server/elasticsearch/elasticsearch_config.test.ts

```diff
@@ -23,19 +23,32 @@ import {
   mockReadPkcs12Truststore,
 } from './elasticsearch_config.test.mocks';
 
-import { ElasticsearchConfig, config, ElasticsearchConfigType } from './elasticsearch_config';
-import { loggingServiceMock } from '../mocks';
-import { Logger } from '../logging';
-
-const createElasticsearchConfig = (rawConfig: ElasticsearchConfigType, log?: Logger) => {
-  if (!log) {
-    log = loggingServiceMock.create().get('config');
-  }
-  return new ElasticsearchConfig(rawConfig, log);
+import { ElasticsearchConfig, config } from './elasticsearch_config';
+import { applyDeprecations, configDeprecationFactory } from '../config/deprecation';
+
+const CONFIG_PATH = 'elasticsearch';
```

Comment:
Is my assumption correct that we don't want to use `config.path` here because we want the tests to ensure this is being set correctly?

Reply:
Nope. I modeled these tests after `core_deprecations.test.ts`; in that file, that the path is hard-coded.

I think hard-coding it here or using `config.path` both make sense.

Reply:
Both are fine imho